siem_integrations/README.md

```diff
@@ -28,8 +28,92 @@ In order to utilize splunk2kafka, a [running Kafka instance](https://kafka.apach
 
 ## CLX Query
 
-Coming soon...
+### Overview
+CLX Query is a [Splunk](https:/www.splunk.com) application that requests CLX Query Service ([Django](https:/www.djangoproject.com) RESTful service) to execute queries using CLX python package and get results back to the Splunk GUI. CLX python package uses [BlazingSQL](https://blazingsql.com) engine internally to execute queries. [Gunicorn](https:/gunicorn.org) provides Web Server Gateway Interface to forward requests to CLX Query Service. [Supervisor](http:/supervisor.org) monitors Gunicorn Web Server to ensure it doesn't go offline.
 
+### Requirements
+Install required dependencies if not available.
+
+#### Install from Source
+```aidl
+pip install django==3.0.3 gunicorn==20.0.4 djangorestframework==3.11.0 supervisor==4.1.0
+```
+#### Conda Install
+```aidl
+conda install -c conda-forge django=3.0.3 gunicorn=20.0.4 djangorestframework=3.11.0 supervisor=4.1.0
+```
+
+### Download sample dataset
+Download MovieLens stable benchmark dataset. It has 25 million ratings and one million tag applications applied to 62,000 movies by 162,000 users. Includes tag genome data with 15 million relevance scores across 1,129 tags. 
```

Comment:
Provide a link to the MovieLens dataset you mention

https://grouplens.org/datasets/movielens/25m/